templates/macros/menu.html

```diff
@@ -38,11 +38,43 @@
             <ul class="menu__inner">
             {%- for item in menu_items %}
                 <li {%- if current_item and current_item == item %} class="active" {%- endif %}>
-                    {%- if item.newtab -%}
-                        <a href="{{ item.url | replace(from="$BASE_URL", to=config.base_url) | safe }}" target="_blank" rel="noopener noreferrer">{{ item.name | safe }}</a>
+
+                    <!-- RSS -->
+                    {%- set is_rss = item.url == "$BASE_URL/$FEED_FILENAME" -%}
+
+                    {%- if config.extra.rss_icon_color %}
+                        {%- set rss_icon_color = config.extra.rss_icon_color %}
+                    {%- else %}
+                        {%- set rss_icon_color = "#ee802f" %}
+                    {%- endif %}
+
+                    {%- if is_rss -%}
+                        {%- if config.generate_feed %}
+                            {%- if config.feed_filename == "rss.xml" %}
+                                {% set feed_type = 'rss+xml' %}
+                            {%- else %}
+                                {% set feed_type = 'atom+xml' %}
+                            {% endif -%}
+
+                            <a type="application/{{feed_type}}" title="RSS" href="{{ get_url(path=config.feed_filename) | safe }}">
+                                <svg id="rss-icon" xmlns="http://www.w3.org/2000/svg" width="22" height="22" viewBox="0 0 32 32" fill="none" stroke="{{rss_icon_color}}" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-rss">
```

Comment:
We should give credit to feather icons https://feathericons.com/ and include their LICENSE file in the repo https://github.com/feathericons/feather/blob/main/LICENSE as `LICENSE-Feather.md` (similar to Hack fonts).

Reply:
Added in https://github.com/pawroman/zola-theme-terminimal/pull/63/commits/5017bf67a0ad577cd6fad6ffe05701e783583ec5